Xamarin.Forms.Core/Properties/GlobalAssemblyInfo.cs

```diff
@@ -1,6 +1,35 @@
 using System.Reflection;
 
+#if DEBUG
+[assembly: AssemblyConfiguration("DEBUG")]
+#endif
+#if RELEASE
+[assembly: AssemblyConfiguration ("RELEASE")]
+#endif
+
 [assembly: AssemblyCompany("Xamarin Inc.")]
 [assembly: AssemblyProduct("Xamarin.Forms")]
-[assembly: AssemblyCopyright("Copyright � Xamarin Inc. 2013-2017")]
+[assembly: AssemblyCopyright("Copyright � Xamarin Inc. 2013-2019")]
 [assembly: AssemblyTrademark("")]
+#pragma warning disable CS0436  // Type conflicts with imported type
+
+[assembly: AssemblyVersion(
```

Comment:
This usage of the direct attributes completely negates the whole purpose of the `GetVersion` target you have above.